widgets/SFTPCopy-action.json

```diff
@@ -97,6 +97,18 @@
             "key-placeholder": "key",
             "value-placeholder": "value"
           }
+        },
+        {
+          "widget-type": "select",
```

Comment:
For boolean attributes, we usually use the toggle type. See https://github.com/cdapio/hydrator-plugins/blob/develop/core-plugins/widgets/File-batchsource.json#L80

Reply:
Fixed